man/homectl.xml

```diff
@@ -929,6 +929,27 @@
 
         <xi:include href="version-info.xml" xpointer="v245"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>--session=</option><replaceable>SESSION</replaceable></term>
+
+        <listitem><para>Takes a string argument. Configures the user's preferred session
+        (i.e. <literal>gnome</literal>, <literal>plasma</literal>, and so on). This is read by
+        the display manager to pick the default session the user is logged into.</para>
```

Comment:
the reference to XDG_SESSION_DESKTOP/XDG_CURRENT_DESKTOP should be added here somewhere, so that people don't make up their own names, but reuse the ones from those variables.

Reply:
As mentioned elsewhere. This != the variables